mozilla-release/browser/installer/windows/nsis/shared.nsh

```diff
@@ -819,334 +799,6 @@ ${EndIf}
 !macroend
 !define RemoveDeprecatedKeys "!insertmacro RemoveDeprecatedKeys"
 
-<<<<<<< HEAD
-; Resets Win8+ specific toast keys Windows sets. We call this on a
-; fresh install and on uninstall.
-!macro ResetWin8PromptKeys KEY PREFIX
```

Comment:
We have separate task to check everything related to installer (a lot of changes was in this version)